Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -506,7 +506,36 @@ public override void StartDriver()
                         {
                             ChService.HideCommandPromptWindow = HideConsoleWindow;
                         }
-                        Driver = new ChromeDriver(ChService, options, TimeSpan.FromSeconds(Convert.ToInt32(HttpServerTimeOut)));
+
+                        try
+                        {
+                            Driver = new ChromeDriver(ChService, options, TimeSpan.FromSeconds(Convert.ToInt32(HttpServerTimeOut)));
+                        }
+                        catch (Exception ex)
+                        {
+                            //If the os is alpine linux
+                            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && ex.Message.ToLower().Contains("no such file or directory"))
+                            {
```

Comment:
add a check if the path for chromium and driver exists if yes then only try to restart the driver else 
throw the exception

Reply:
Done